// Copyright 2025 Google LLC
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto3";

package google.confidential_match.lookup_server.proto_backend;

option java_package = "com.google.cm.lookupserver.backend";
option java_outer_classname = "ExportMetadataProto";

// Represents the export metadata file created as part of a data export.
message ExportMetadata {
  // A base-64 encoded encrypted DEK used to protect the data in the export.
  string encrypted_dek = 1;
}
